.mergify.yml

```diff
@@ -34,6 +34,25 @@ queue_rules:
               - "status-success=golangci-lint"
               - "status-success=mod-check"
               - "status-success=lint-extras"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.26"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.27"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.28"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.26"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.27"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.28"
+              - "status-success=ci/centos/mini-e2e/k8s-1.26"
+              - "status-success=ci/centos/mini-e2e/k8s-1.27"
+              - "status-success=ci/centos/mini-e2e/k8s-1.28"
+              - "status-success=ci/centos/upgrade-tests-cephfs"
+              - "status-success=ci/centos/upgrade-tests-rbd"
+          - and:
+              - base~=release-v3.[89]
```

Comment:
let's ignore release v3.8 since it will be deprecated when 3.10 comes out ?

Reply:
sure, removed it.